src/libs/settingTools.js

```diff
@@ -0,0 +1,28 @@
+'kiwi public';
+
+let ignoreKeys = ['emojis', 'themes', 'bnc', 'aliases', 'restricted', 'kiwiServer',
+    'hide_advanced', 'windowTitle', 'startupOptions', 'plugins', 'presetNetworks'];
+
+export function buildTree(data, base, object, modified) {
```

Comment:
This could do with a comment explaining what it does now that there is no context around it